src/python/WMCore/FwkJobReport/Report.py

```diff
@@ -192,14 +192,16 @@ def jsonizePerformance(perfSection):
         Convert the performance section of the FWJR into JSON.
         """
         jsonPerformance = {}
-        for reportSection in ["storage", "memory", "cpu", "multicore"]:
+        for reportSection in ["storage", "memory", "cpu", "multicore", "cmssw"]:
             jsonPerformance[reportSection] = {}
             if not hasattr(perfSection, reportSection):
                 continue
 
             jsonPerformance[reportSection] = getattr(perfSection, reportSection).dictionary_()
             for key in jsonPerformance[reportSection]:
                 val = jsonPerformance[reportSection][key]
+                if reportSection == 'cmssw' and isinstance(val, ConfigSection):
+                    jsonPerformance[reportSection][key] = val.dictionary_()
                 if isinstance(val, float):
```

Comment:
should this line become an `elif` to save some cpu cycles?

Reply:
literally it will not make any savings to cpu cycles as such effect is negligible, but since you asked for it I'll make a change.

Reply:
done